devlib/instrument/daq.py

```diff
@@ -55,6 +55,7 @@ def __init__(self, target, resistor_values,  # pylint: disable=R0914
                                                  resistor_values=resistor_values,
                                                  channel_map=channel_map,
                                                  labels=labels)
+        self.sample_rate_hz = sampling_rate
```

Comment:
We're still in version 0, so backward compatibility is not a hard guarantee yet. It's better to fix these things now, while we have the chance.